lib/pbench/agent/tool_data_sink.py

```diff
@@ -1254,9 +1263,11 @@ def execute(self):
             self._tm_tracking = self.record_tms(tms)
             self._num_tms = len(self._tm_tracking.keys())
 
-            # Tell the entity that started us who we are, indicating we're
-            # ready, and all the TMs are ready.
-            started_msg = dict(kind="ds", action="startup", status="success")
+            status = check_failed_tool_init(tms)
```

Comment:
As per https://github.com/riya-17/pbench/pull/5, we do not want to have the Tool Data Sink applying a policy based on failed tools.  That policy should be applied by `pbench-tool-meister-start` where it can direct what the Tool Data Sink should do as a result.